Samples/Swift/DaysUntilBirthday/Shared/Views/VerificationView.swift

```diff
@@ -52,6 +62,16 @@ struct VerificationView: View {
                  action:{refresh(results: result)})
         }
       }
+      .sheet(isPresented: $verifiedAgeViewModel.isShowingAgeVerificationSignal) {
+          AgeVerificationResultView(ageVerificationSignal: verifiedAgeViewModel.ageVerificationSignal)
+      }
+      .alert(isPresented: $verifiedAgeViewModel.isShowingAgeVerificationAlert, content: {
```

Comment:
I think this is deprecated: https://developer.apple.com/documentation/swiftui/view/alert(ispresented:content:)

Reply:
Updated.